src/dosbox.cpp

```diff
@@ -666,15 +666,17 @@ void DOSBOX_Init()
 	pstring->Set_help("CPU core used in emulation ('auto' by default). 'auto' will switch to dynamic\n"
 	                  "if available and appropriate.");
 
-	const char* cputype_values[] = { "auto", "386", "386_slow", "486_slow", "pentium_slow",
-#if C_MMX
-	                                  "pentium_mmx",
-#endif
-	                                  "386_prefetch", nullptr};
-	pstring = secprop->Add_string("cputype", always, "auto");
+	const char* cputype_values[] = { "386", "486", "pentium", "pentium_mmx",
+	                                 "386_prefetch", "auto", nullptr};
```

Comment:
I think this order would be better:

auto, 386, ..., 486, ..., pentium, ...

Reply:
OK, changed.